docs/overrides/javascript/extra.js

```diff
@@ -26,9 +36,11 @@ function checkAutoTheme() {
     }
   }
 }
-// ! No need to run the function when the script loads as by default the theme is determined by the user's preference (if auto theme is enabled)
-// checkAutoTheme();
-// Run the function when the user's preference changes (when the user changes their system theme)
+
+// Run function when the script loads
+checkAutoTheme();
```

Comment:
@glenn-jocher We need to run the function when the script loads - without this, if my system is on Dark Mode, the default theme will be light (on initial load).

Reply:
Yes I noticed this too!